client/solve.go

```diff
@@ -415,21 +415,21 @@ func parseCacheOptions(opt SolveOpt) (*cacheOptions, error) {
 				logrus.Warning("local cache import at " + csDir + " not found due to err: " + err.Error())
 				continue
 			}
-			// if digest is not specified, load from "latest" tag
+			// if digest is not specified, load from something tag
```

Comment:
The default tag should be still "latest". Custom tag should be specified by `attr["tag"]`.

Reply:
Thank you for pointing out. I had a misunderstanding. (Non-latest support => custom tag support)
When a custom tag was passed, I added a process to compare with `m.Annotations [ocispec.AnnotationRefName]`.